lib/hanrahashiv.py

```diff
@@ -15,28 +14,34 @@ class FormulaModifications(Scene):
         formula = Tex('a', '$\cdot$', 'b')
     """
 
-    def rearrange_formula(self, formula : Tex or MathTex, new_sequence : list, move_up : list, move_down : list):
+    def rearrange_formula(self,
+        formula : Tex or MathTex, new_sequence : list, 
+        move_up : list, move_down : list,
+        fade_out : list, fade_in : list
+    ):
         tex_string_list = [tex.get_tex_string() for tex in formula]
         new_tex_string_list = [tex_string_list[new_sequence[i]] for i in range(len(tex_string_list))]
         new_formula = Tex(*new_tex_string_list, font_size=formula[0].font_size, color=formula.color).move_to(formula)
 
         in_line = []
         for i in range(len(tex_string_list)):
-            if i not in move_up and i not in move_down:
+            if i not in move_up and i not in move_down: #and i not in fade_out: # i not in fade_in and
                 in_line.append(i)
 
         self.play(
             *[formula[i].animate.shift(0.5 * UP) for i in move_up],
-            *[formula[i].animate.shift(0.5 * DOWN) for i in move_down]
+            *[formula[i].animate.shift(0.5 * DOWN) for i in move_down],
+            *[formula[i].animate.set_opacity(0) for i in fade_out]
```

Comment:
Why not FadeOut? Also, do you need to remove those from `new_formula` and `new_tex_string_list`?

Reply:
ֆեյդաութ որ անում եմ, հետո ընդեղ տրանսֆորմ անելուց ապուշավարի բան ա ստացվում, սենց որ օփասիթին 0 եմ դնում ու շիֆթ եմ անում move_up-ների հետ, սաղ լավ ա լինում։
չջոկեցի, ի՞նչ ռիմուվ